.github/workflows/tag-for-release.yml

```diff
@@ -233,25 +233,25 @@ jobs:
           ./COMMIT-ID
           ./SHA256.sum
 
-  trigger-tests:
-    needs: [tag-uberjar-for-release]
-    permissions:
-      actions: write
-    runs-on: ubuntu-22.04
-    timeout-minutes: 5
-    steps:
-      - name: trigger pre-release testing
-        uses: actions/github-script@v7
-        with:
-          script: | # js
-            github.rest.actions.createWorkflowDispatch({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              workflow_id: 'pre-release.yml',
-              ref: '${{ github.ref }}',
-              inputs: {
-                commit: '${{ inputs.commit }}',
-                version: '${{ inputs.version }}',
-                auto: '${{ inputs.auto }}',
-              }
-            });
+  run-pre-release-tests:
+    name: Run pre-release tests for ${{ inputs.version }}
+    needs: tag-uberjar-for-release
+    uses: ./.github/workflows/pre-release.yml
+    secrets: inherit
+    with:
+      commit: ${{ inputs.commit }}
+      version: ${{ inputs.version }}
+
+  auto-publish-release:
+    name: Auto-publish release ${{ inputs.version }}
+    if: ${{ inputs.auto }}
+    needs: run-pre-release-tests
+    uses: ./.github/workflows/release.yml
+    secrets: inherit
+    with:
+      version: ${{ inputs.version }}
+      commit: ${{ inputs.commit }}
+      auto: ${{ inputs.auto }}
+      # for now don't auto-set release channel because we don't want to have
+      # multiple nightlies due to race conditions
+      release-channel: none
```

Comment:
Aha! The answer to [my question](https://github.com/metabase/metabase/pull/48633#discussion_r1828131879) from the other PR.